### ISSUE TITLE
Installation via pip not working?

### ISSUE THREAD

**Issue by nomeata** [cptpcrd/pyseccomp#1]
It seems that `pip install` doesn't actually install any `.py` files, only the `dist-info`:
```
$ pip install pyseccomp==0.1.1
Requirement already satisfied: pyseccomp==0.1.1 in ./env/lib/python3.7/site-packages (0.1.1)
$ find env/**/*pyseccomp*
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/RECORD
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/WHEEL
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/INSTALLER
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/LICENSE
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/METADATA
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/top_level.txt
env/lib/python3.7/site-packages/pyseccomp-0.1.1.dist-info/REQUESTED
$ python3 -m pyseccomp
…/env/bin/python3: No module named pyseccomp
```

**Comment by nomeata** [comment]
This seems to help:
```diff
diff --git a/setup.cfg b/setup.cfg
index f31f393..596cfd2 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -15,6 +15,7 @@ classifiers =
     Operating System :: POSIX :: Linux
 
 [options]
+py_modules = pyseccomp
 packages = find:
 python_requires = >=3.6
 
```

**Comment by cptpcrd** [comment]
> This seems to help:
> 
> ```diff
> diff --git a/setup.cfg b/setup.cfg
> index f31f393..596cfd2 100644
> --- a/setup.cfg
> +++ b/setup.cfg
> @@ -15,6 +15,7 @@ classifiers =
>      Operating System :: POSIX :: Linux
>  
>  [options]
> +py_modules = pyseccomp
>  packages = find:
>  python_requires = >=3.6
>  
> ```

Yes, some research seems to indicate that will work. I'll do some testing and release a fix together with #2.

**Comment by nomeata** [comment]
Thanks! Although it may be that I can't use this after all, it seems that Amazon Lambda doesn't support seccomp :-(
